lib/thrift/generator/server/binary_framed.ex

```diff
@@ -94,6 +95,7 @@ defmodule Thrift.Generator.Server.BinaryFramed do
       Thrift.TApplicationException ->
         quote do
           unhandled ->
+            Logger.error("Unhandled exception: #{Exception.message(unhandled)}")
```

Comment:
This just logs the message, right? I think we could get the stack trace too with Exception.format.